MANIFEST.in

```
@@ -1,2 +1,12 @@
-include LICENSE
+prune .github*
+prune doc*
+prune examples*
+prune icepyx/tests*
+exclude *.yml
+exclude *.yaml
+exclude *.ini
+exclude .coveragerc
+exclude .mailmap
+exclude .gitignore
+exclude *.txt
```

```suggestion
exclude .coveragerc
exclude .gitignore
exclude .mailmap
exclude *.ini
exclude *.txt
exclude *.yaml
exclude *.yml
```
Just sorting this alphabetically :)

---

Thanks! :smile: